src/app/leverage/components/selectors/network-selector.tsx

```diff
@@ -42,7 +42,7 @@ export function NetworkSelector(props: NetworkSelectorProps) {
           onClick={() => {
             props.onSelectNetwork(BASE.chainId)
           }}
-        />
+        /> */}
```

Comment:
Hiding Base until we need it. Adding Mainnet in first position according to extra ticket.